pkgs/servers/xmpp/ejabberd/default.nix

```diff
@@ -70,7 +71,7 @@ in stdenv.mkDerivation rec {
 
     outputHashAlgo = "sha256";
     outputHashMode = "recursive";
-    outputHash = "sha256-Lj4YSPOiiJQ6uN4cAR+1s/eVSfoIsuvWR7gGkVYrOfc=";
+    outputHash = "sha256-v/0lK+7Pt0F33ZU2mQIoUH9JHHeL5WdtbxiRIjqq0QM=";
```

Comment:
This hash is not reproducible because of the typo here https://github.com/NixOS/nixpkgs/blob/95645f522d389f1c6db6026c20ea6f3e901c1aba/pkgs/servers/xmpp/ejabberd/default.nix#L62

Try to replace `.got` with `.git`, and test locally that the `.got` file doesn't actually exist for example `if test -e deps/.got; then echo it exists; exit 1; fi`

Reply:
Remember that a build is skipped if the hash is found in the store, so always empty the hash when editing inside a fixed output derivation (FOD).